docs/reference/release-cycle.md

```diff
@@ -0,0 +1,109 @@
+be required to upgrade their version for up to 1.5 years. The Crossplane
+community also supports the latest release, whether it is an LTS release or not.
+
+![Release Diagram](../media/crossplane-releases.png)
+
+For example, Crossplane v1.0.0 was the first LTS release, so the next one will
+be v1.1.0 in February 2021. At that time, the Crossplane community will support
+both v1.0.0 and v1.1.0. When v1.2.0 is released in April 2021, the community
+will then support v1.0.0 and v1.2.0. The next release, v1.3.0, will be an LTS
+release. The Crossplane community will support v1.0.0 and v1.3.0 until the
+v1.4.0 release, at which point it will support all three. This pattern would
+continue until the release of v1.9.0, at which time support for v1.0.0 will be
+dropped.
+
+This means that the Crossplane community will support at most four releases
+(three LTS + one interim) at at a time.
+
+### Definition of Support
+
+The Crossplane community defines support in that relevant bug fixes that are
+merged to the main development branch will be eligible to be back-ported to the
+release branch of any currently supported version, and patch releases will be
+cut appropriately. It is also possible that a fix may be merged directly to the
+release branch if no longer applicable on the main development branch. Support
+does not indicate any SLA on response time for user support in the form of Slack
+messages or issues, but problems will be addressed on a best effort basis by
+maintainers and contributors for currently supported releases.
+
+### Patch Releases
+
+_This policy is subject to change in the future._
+
+Patch releases are cut for currently supported minor versions on an as-needed
+basis. Any critical back-ported fixes will be included in a patch release as
+soon as possible after merge.
+
+### Pre-Releases
+
+_This policy is subject to change in the future._
+
+Alpha, Beta, and RC releases are cut for an upcoming release on an as-needed
+basis. As a policy, at least one pre-release will be cut prior to any minor
+release. Pre-releases will not be made on release branches.
+
+### Provider Releases
+
+The Crossplane release process is not required to be adhered to by any other
+Crossplane projects, but a similar cadence is encouraged. Maintainers listed in
+each repositories `OWNERS.md` file are responsible for determining and
+publishing the release process for their project.
+
+## Release Stages
+
+The following stages are the main milestones in a Crossplane release.
+
+### Active Development
+
+During active development, any code that meets the requisite criteria (i.e.
+passing appropriate tests, approved by a maintainer, etc.) will be merged into
+the main development branch. At present, there is no requirement to formally
+submit an enhancement proposal prior to the start of the release cycle, but
+contributors are encouraged to open an issues and gather feedback before
+starting work on a major implementation.
+
+### Feature Freeze
```

Comment:
Do you have any ideas around how we can keep ourselves accountable for not violating the feature and code freezes? Perhaps a label we can use if and when we make exceptions?

Reply:
I think that is a good idea. I made sure to make this document indicate that our exception process is mostly informal right now because it doesn't make a ton of sense to implement a hard line before we really understand how the community will function with contributors / maintainers at large scale. We should eventually have some sort of automated merge blocker that requires an explicit invocation to allow an exception to go through in feature / code freeze.

Reply:
Another thing we should consider in the future it whether we want to go ahead and cut release branch at feature freeze and then backport any fixes. This would fall more under [release process](https://crossplane.io/docs/master/contributing/release-process.html) than release cycle though, and I am not sure it is worth the extra effort at this point with our current flow of contributions.